src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import java.util.HashMap;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(HashMap<String, Integer> storage, FruitTransaction transaction) {
```

Comment:
The 'storage' should be encapsulated. Directly using 'HashMap<String, Integer>' here exposes the implementation details of the storage. Consider creating a separate class to handle storage operations.